app/main.py

```diff
@@ -1,8 +1,673 @@
-from fastapi import FastAPI
```

Comment:
app/main.py is an entry point of the app and should not contain all the routes and APIs implementations, but just imports and only things important for running up the app. So as mentioned before, try to implement MVC pattern and mode all these functions in matching folders and files. You can implement routers for each route according to the models you have in the project. For each route implement api calls that should be under that route. For each router, you should have a controller doing the logic. At the end of the chain, there is a model that the controller is manipulating.